benchmark/mnist_benchmark_visualiser.py

```diff
@@ -0,0 +1,95 @@
+# © Crown Copyright GCHQ
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Visualise the results of ``mnist_benchmark.py``."""
+
+import json
+
+import numpy as np
+from matplotlib import pyplot as plt
+
+
+def main() -> None:
+    """Load benchmark results and visualise the algorithm performance."""
+    with open("mnist_benchmark_results.json", "r", encoding="utf-8") as file:
```

Comment:
Similarly for blobs, enforce the location of the .json file. Suggested change: 
     
     base_dir = os.path.dirname(os.path.abspath(__file__))  # Gets the directory of current script
    # Save results to JSON file
    with open(os.path.join(base_dir,"mnist_benchmark_results.json"), "w", encoding="utf-8") as f: